docs/book/src/SUMMARY.md

```diff
@@ -58,17 +58,20 @@
         - [Multi-tenancy](./developer/architecture/controllers/multi-tenancy.md)
         - [Support multiple instances](./developer/architecture/controllers/support-multiple-instances.md)
     - [Provider Implementers](./developer/providers/implementers.md)
-        - [v0.3 to v0.4](./developer/providers/v0.3-to-v0.4.md)
-        - [v0.4 to v1.0](./developer/providers/v0.4-to-v1.0.md)
-        - [v1.0 to v1.1](./developer/providers/v1.0-to-v1.1.md)
-        - [v1.1 to v1.2](./developer/providers/v1.1-to-v1.2.md)
-        - [Cluster Infrastructure](./developer/providers/cluster-infrastructure.md)
-        - [Machine Infrastructure](./developer/providers/machine-infrastructure.md)
-        - [Bootstrap](./developer/providers/bootstrap.md)
+        - [Version migration](./developer/providers/version-migration.md)
```

Comment:
```suggestion
        - [Migration guides](./developer/providers/version-migration.md)
```
nit, if you want (same for the file name)

Reply:
I think version migration is clearer and more specific here, but if there's a call to use the more generic title I'll change it.

Reply:
Just keep it as is. Version migration just sounded strange to me, but my entire understanding of the english language is subjective :)

Reply:
As is mine!

Reply:
I hope yours is better than mine and you correct the nits I drop on PRs of other folks if they are just too wrong :)